agent/agent.go

```diff
@@ -422,7 +444,11 @@ func (a *Agent) runPostGroupCommands(ctx context.Context, tc *taskContext) {
 	if tc.taskConfig == nil {
 		return
 	}
-	a.resetLogging(ctx, tc)
+	err := a.resetLogging(ctx, tc)
+	if err != nil {
+		grip.Critical(errors.Wrap(err, "error making post-group logger"))
```

Comment:
I'm concerned that `resetLogging` is already throwing an error, something having to do with closing the logger harness. Does grip.Critical go to splunk? If so, we may wish to log at a lower level, or fix the grip bug, if there is one.

Reply:
The one I'm currently aware of is that we sometimes close the agent's log file when it's already closed, but that will currently log and not return an error (line 268). Is there something else you're thinking of? I believe sending errors here to splunk is appropriate since we won't know about it otherwise (unless we just don't care about this error that much)

Reply:
I think it's fine as is then. If we need to back out of the change, that's fine.